### DISCUSSION TITLE
갑자기 config.yaml이 안 먹는 듯합니다.

### DISCUSSION THREAD

**Discussion by woncheol-kim** [Riey/kime#605] [Q&A] [answered]
kime 자체는 작동은 하는 것 같은데, 모든 설정값이 디폴트로 작동합니다.
한영 전환 단축키가 오른쪽 Alt로 고정된 것은 둘째치고, 
한글 레이아웃이 두벌식으로 바뀌어서 세벌식 쓰는 입장에서 너무너무 불편하네요.

처음부터 이랬던 건 아니고, 그동안 잘 되다가 최근에 아치리눅스 소프트웨어 패키지 업데이트한 뒤로 이렇네요.

OS: EndeavourOS Linux x86_64
Kernel: 6.0.11-arch1-1

[config.yaml]
  hangul:
    layout: sebeolsik-3-90
    word_commit: false
    addons:
      all:
        - ComposeChoseongSsang
      dubeolsik:
        - TreatJongseongAsChoseong

혹시 몰라서, config.yaml 내용은 가능한 모든 곳에 동일하게 적용했습니다.
/usr/share/doc/kime/default_config.yaml
/etc/xdg/kime/config.yaml
~/.config/kime/config.yaml

$ echo $XDG_CONFIG_HOME
(blank)
$ echo $XDG_CONFIG_DIR
(blank)

**Comment by honnip** [comment, marked as answer]
v3.0.0에서 설정 파일에 변경이 있습니다. [res/default_config.yaml](https://github.com/Riey/kime/blob/645aabeb5c910a17745717024ca3abd4a0640c94/res/default_config.yaml)

**Reply by woncheol-kim** [reply]
해결됐습니다. 감사합니다.
